src/pages/locations/[id].tsx

```diff
@@ -53,30 +54,33 @@ const LocationDetails = () => {
   }
 
   return (
-    <div className="flex flex-col gap-8">
-      {/* TODO: Create shared component for Header */}
-      <Header title={location.name} label="Location" />
-      <div className="flex flex-col gap-4">
-        <div
-          className="bg-slate-200 rounded"
-          style={{
-            width: `${pageWidth - 32}px`,
-            height: `${pageWidth - 32}px`,
-          }}
-        >
-          <img src={location.imageUrl} />
-        </div>
-        <div className="flex flex-col gap-4 jus">
-          <div className="flex flex-col">
-            <Label>Description</Label>
-            <Description>{location.description}</Description>
+    <div>
+      <AppBackHeader />
```

Comment:
This leads to an infinite loop where it takes you back to the tap page and it says you've already visited. This should link directly to social tab

Reply:
https://github.com/nfc-denver/nfc-denver/issues/68